app/src/main/java/org/android/go/sopt/di/RetrofitModule.kt

```diff
@@ -0,0 +1,74 @@
+package org.android.go.sopt.di
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.android.go.sopt.BuildConfig
+import org.android.go.sopt.data.type.BaseUrlType
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object RetrofitModule {
+    private const val SIGN_UP_BASE_URL: String = BuildConfig.SIGN_UP_BASE_URL
+    private const val REQRES_BASE_URL: String = BuildConfig.REQRES_BASE_URL
```

Comment:
```suggestion
import org.android.go.sopt.BuildConfig.SIGN_UP_BASE_URL
import org.android.go.sopt.BuildConfig.REQRES_BASE_URL
```

상수값 임포트하면 따로 선언 안 해줘도 되는 변수들인 것 같아여

Reply:
천재다